src/components/NcDashboardWidgetItem/NcDashboardWidgetItem.vue

```diff
@@ -31,7 +31,7 @@ This component is meant to be used inside a DashboardWidget component.
 <template>
 	<div @mouseover="hovered = true" @mouseleave="hovered = false">
 		<component :is="targetUrl ? 'a' : 'div'"
-			:href="targetUrl"
+			:href="targetUrl || undefined"
```

Comment:
So this will then remove the href attribute from the component?

Reply:
yes. Attributes with `null` or `undefined` will be removed.

Reply:
great!